common/src/main/java/com/microsoft/identity/common/logging/DiagnosticContext.java

```diff
@@ -22,39 +22,21 @@
 // THE SOFTWARE.
 package com.microsoft.identity.common.logging;
 
-public class DiagnosticContext {
+import com.microsoft.identity.common4j.external.logging.IRequestContext;
```

Comment:
What does the external package name signify?

Reply:
External means it will be consumed externally (i.e. by android packages)

Reply:
Change the scheme to match with android-common (anything internal will be under /internal, anything else is external)